### PR TITLE
[WNMGDS-2522] Remove the unused "success" text field

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

I ran a scan of our dependent projects to make sure it wasn't actually being used by any products. The only places it was referenced was in copies of our design system code.

- Removed the `ds-c-field-success` class and all references to it
- Removed the associated tokens
- Removed the associated story
- Removed the VRT refs

Screenshot of the thing we're removing:

Screenshot of a text field with a green 'success' border

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title` or [NO-TICKET] if this is unticketed work.
- [x] Selected appropriate `Type` (only one) label for this PR, if it is a breaking change, label should only be `Type: Breaking`
- [x] Selected appropriate `Impacts`, multiple can be selected.
- [x] Selected appropriate release milestone

### CODE REVIEW

File: packages/design-system-tokens/src/themes/cmsgov-component.ts [hunk 1]
```diff
@@ -410,13 +410,11 @@ export const components: AnyTokenValues = {
     '__background-color--disabled':               t.color['gray-lighter'],
     '__border-width':                             '2px',
     '__border-width--error':                      '3px',
-    '__border-width--success':                    '3px',
     '__border-color':                             t.color['base'],
     '__border-color--disabled':                   t.color['gray-light'],
     '__border-color--error':                      t.color['error'],
     '__border-color--error--inverse':             t.color['error-light'],
     '__border-color--inverse':                    t.color['black'],
-    '__border-color--success':                    t.color['success-light'],
```
Comment (zarahzachz) [inline packages/design-system-tokens/src/themes/cmsgov-component.ts:419]: do we still need the `success-light` color token?

Comment (zarahzachz) [reply packages/design-system-tokens/src/themes/cmsgov-component.ts:419]: nevermind! just remembered we use this for success alarts and badges and stuff

Comment (pwolfert) [reply packages/design-system-tokens/src/themes/cmsgov-component.ts:419]: We're still using it in other places, and we made a push a while back to have a full gradient of each of our color families even if we don't necessarily use them all (like `-lightest` to `-darkest`)

**Review by zarahzachz — APPROVED** [review]

**Review by brentschneider — APPROVED** [review]
Cool, cool — I'll spin up a ticket to do the same in the UI Kits

**Comment by hi-its-hailey** [comment]
Looks good to me!

**Pull request merged**